deep_qa/models/reading_comprehension/gated_attention_reader.py

```diff
@@ -152,7 +159,7 @@ def _build_model(self):
             # (batch size, document length, question length)
             normalized_qd_attention = MaskedSoftmax()([qd_attention])
 
-            gated_attention_layer = GatedAttention()
+            gated_attention_layer = GatedAttention(self.gating_function)
```

Comment:
Looks like you should give this a name.  This is why the theano test is failing.